examples/log-errors/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <button id="err-btn" onclick="throwError();">Error!</button>
+    <button id="warn-btn" onclick="throwWarning();">Warning!</button>
+
+    <script>
+        function throwError() { 
+            var error = new Error("This is a 'critical' error");
+            error.level = 'critical';
+            error.url   = 'https://sub-domain.example.com';
+            console.log(error.level)
```

Comment:
Add a new line before and after `console.log` statement because it's a different logical block.